src/ray/core_worker/actor_handle.cc

```diff
@@ -45,7 +45,7 @@ ray::rpc::ActorHandle CreateInnerActorHandleFromString(const std::string &serial
 }
 
 ray::rpc::ActorHandle CreateInnerActorHandleFromActorTableData(
-    const ray::gcs::ActorTableData &actor_table_data) {
+    const ray::rpc::ActorTableData &actor_table_data) {
```

Comment:
Could you explain why we need this kind of change?

Reply:
because xxxTableData is originally in namespace of `ray::rpc`, we use `ray::gcs` because we give it a alias name in `tables.h` before.